crates/data/scripts/enemy-parser/enemy-parser.js

```diff
@@ -0,0 +1,418 @@
+const fs = require('node:fs')
+
+const enemy_locale_kv = require('./locale/Enemies-CAB-18cc0a97ed3fba0abaeabffc8cbc6c92-8020956684977417460.json')
+const enemy_locale_definition = require('./locale/Enemies_EN-CAB-18cc0a97ed3fba0abaeabffc8cbc6c92--9219550977792058189.json')
+const character_locale_kv = require('./locale/Characters-CAB-18cc0a97ed3fba0abaeabffc8cbc6c92--592314051786654548.json')
+const character_locale_definition = require('./locale/Characters_EN-CAB-18cc0a97ed3fba0abaeabffc8cbc6c92--5266921922541985654.json')
+
+let enemy_pairs = enemy_locale_kv.locIndexByLocStringId.keys.Array.map((val, i) => {
+  return [val, enemy_locale_kv.locIndexByLocStringId.values.Array[i]]
+})
+
+let character_pairs = character_locale_kv.locIndexByLocStringId.keys.Array.map((val, i) => {
+  return [val, character_locale_kv.locIndexByLocStringId.values.Array[i]]
+})
+
+const enemy_locale = Object.fromEntries(enemy_pairs)
+const character_locale = Object.fromEntries(character_pairs)
+
+const getFromLocale = (name) => {
+  let eidx = enemy_locale[name]
+  let enemy = enemy_locale_definition.strings.Array[eidx]
+
+  let cidx = character_locale[name]
+  let character = character_locale_definition.strings.Array[cidx] 
+  return enemy || character
+}
+
+let output = ""
+let names = []
+
+// Reset the current output file
+fs.writeFileSync('./output/output.rs', output)
```

Comment:
Could maybe set this with a const instead, since it's used in another place too.